test/unit/html/HtmlConverterTest.php

```diff
@@ -1114,6 +1114,30 @@ public function testConvertToAppropriateBodyForApiWithMultipleComment()
         $this->assertEquals($expected_html, $translated_html);
     }
 
+    public function testConvertToAppropriateBodyForApiWithFormHavingMultipleInputs()
+    {
+        $original_html = '<html><body><head></head>';
+        $expected_converted_html = '<html><body><head></head>';
+        $expected_reverted_html = '<html><body><head></head>';
+
+        for ($i = 0; $i < 50; $i++) {
+            $original_html .= "<input type=\"hidden\" name=\"field_{$i}\" value=\"text-{$i}\">";
```

Comment:
Note: `value` here cannot be `'0'`, otherwise tests would fail, e.g.:
```
--- Expected
+++ Actual
@@ @@
-'<html><body><head></head><input type="hidden" name="field_0" value="__wovn-backend-ignored-key-0"> // ...
+'<html><body><head></head><input type="hidden" name="field_0" value="0"> // ...
```

This is because of:
https://github.com/WOVNio/WOVN.php/blob/77ef83da18b2d6f35a9bd6a065b23c3016e83c27/src/wovnio/html/HtmlConverter.php#L418

in which `!$originalText` returns truthy value.

Reply:
probably worth fixing since numbers are likely to be used for hidden inputs

Reply:
Let's do it on a separate PR?

Reply:
Seems like there's an issue with `getAttribute` (ref. 76efbbd3cc6b2166a587340bf34b55ae995b20a6):
> 2) Wovnio\Wovnphp\Tests\Unit\ModifiedVendor\SimpleHtmlDomTest::testGetAttribute
Failed asserting that false matches expected '0'.

Reply:
if it's not trivial we can do it in a separate PR. But I suspect we can just remove that check entirely and always call strpos. checking for false doesn't seem right.